### ISSUE TITLE
llvmpipe is not enabled for i686

### ISSUE THREAD

**Issue by sham1** [void-linux/void-packages#37349]
### System

* xuname: `Void 5.15.41_1 i686 GenuineIntel uptodate rFF`
  *output of `xuname` (part of xtools)*
* package: `mesa-21.3.7_1`
  *affected package(s) including the version: `xbps-query -p pkgver <pkgname>`*

### Expected behavior

`llvmpipe` should be enabled for i686 targets. It was excluded from getting `llvmpipe` with the commit 0335439623542f562c56b539d508a922be63c132.

### Actual behavior

`llvmpipe` is not enabled for i686 targets.

### Steps to reproduce the behavior

1. Run something using Mesa with software rendering on i686.

**Comment by sham1** [comment]
I can make a PR fixing this, I was just wondering whether this was an intentional omission or if it were just an erroneous exclusion.

**Comment by faithanalog** [comment]
i believe this is probably an erroneous exclusion, because llvmpipe works just fine on i686, have been daily driving it for a long time

**Comment by faithanalog** [comment]
would love to this fixed because it affects my poulsbo hardware (intel atom netbook, no working hardware acceleration unless you use a 3.x kernel and a proprietary module/userspace). Me and a few friends use these things on the regular and we either custom compile mesa or use other distributions which have llvmpipe enabled (like debian).

anyways the fix is a one-liner, just

```
diff --git a/srcpkgs/mesa/template b/srcpkgs/mesa/template
index c90500de74..172d9c12ff 100644
--- a/srcpkgs/mesa/template
+++ b/srcpkgs/mesa/template
@@ -32,7 +32,7 @@ fi
 # especially on big endian it's all kinds of broken, and e.g. on
 # 32-bit powerpc it does not work at all, so fall back to softpipe
 case "$XBPS_TARGET_MACHINE" in
-       x86_64*|aarch64*|ppc64le*|arm*) ;;
+       i686*|x86_64*|aarch64*|ppc64le*|arm*) ;;
        *) configure_args+=" -Ddraw-use-llvm=false" ;;
 esac
 ```

**Comment by sham1** [comment]
Seems that this has now been fixed in the commit e507a26b520a29c65f2b18a5b3eb976ddea9034d.

Closing.
